SuperdenseCoding/Tasks.qs

```diff
@@ -40,22 +45,19 @@ namespace Quantum.Kata.SuperdenseCoding {
     // Task 1. Entangled pair
     // Input: An array of two qubits in the |00⟩ state.
```

Comment:
Please update input description to match operation signature.

Reply:
I updated the description. Take a look if you are happy with it.